portal-impl/src/com/liferay/portal/model/impl/LayoutImpl.java

```diff
@@ -1351,7 +1351,7 @@ private LayoutTypePortlet _getLayoutTypePortletClone(
 	private List<PortletPreferences> _getPortletPreferences(long groupId) {
 		List<PortletPreferences> portletPreferences =
 			PortletPreferencesLocalServiceUtil.getPortletPreferences(
-				groupId, PortletKeys.PREFS_OWNER_TYPE_LAYOUT,
+				groupId, PortletKeys.PREFS_OWNER_TYPE_GROUP,
```

Comment:
I agree that it's weird, but as mentioned above it seems to be the way it was designed and changing it like this will probably cause quite a few regressions.